spark/src/main/scala/io/substrait/spark/logical/ToSubstraitRel.scala

```diff
@@ -378,8 +381,24 @@ class ToSubstraitRel extends AbstractLogicalPlanVisitor with Logging {
   private def buildLocalFileScan(fsRelation: HadoopFsRelation): relation.AbstractReadRel = {
     val namedStruct = toNamedStruct(fsRelation.schema)
 
-    val ff = new FileFormat.ParquetReadOptions {
-      override def toString: String = "csv" // TODO this is hardcoded at the moment
+    val format = fsRelation.fileFormat match {
+      case _: CSVFileFormat =>
+        new FileFormat.DelimiterSeparatedTextReadOptions {
+          // default values for options specified here:
+          // https://spark.apache.org/docs/latest/sql-data-sources-csv.html#data-source-option
+          override def getFieldDelimiter: String = fsRelation.options.getOrElse("delimiter", ",")
+          override def getMaxLineSize: Long = 0 // No Spark equivalent. Assign proto default of 0
```

Comment:
this is a bit weird still, but I see that it's a part of the proto, so I don't have better ideas either. I guess only problem would appear if someone uses spark-produced plans in some other system which would support & implement this, but if that happens we can figure it out then, I think.